core/src/main/java/org/apache/accumulo/fate/zookeeper/ServiceLock.java

```diff
@@ -311,22 +311,19 @@ private synchronized void determineLockOwnership(final String createdEphemeralNo
         @Override
         public void process(WatchedEvent event) {
           if (LOG.isTraceEnabled()) {
-            LOG.trace("[{}] Processing event:", vmLockPrefix);
-            LOG.trace("- type  {}", event.getType());
-            LOG.trace("- path  {}", event.getPath());
-            LOG.trace("- state {}", event.getState());
+            LOG.trace("[{}] Processing {}", vmLockPrefix, event);
```

Comment:
Printing the values in one log statement is nice cause of the likelihood of multiple threads. Does the `WatchedEvent` type toString() print the same values?

Reply:
Yes, it does. It looks like "WatchedEvent: " followed by those three values. There aren't newlines, but I figured formatting changes here doesn't really matter, since it's a trace log message, and thus should be rarely seen.